### ISSUE TITLE
Scala.js facades - native default arguments do not compile

### ISSUE THREAD

**Issue by joschua-fink** [lampepfl/dotty#10841]
## Minimized code

```Scala
@js.native
@JSImport("three", "Vector3")
class Vector3(var x: Double = js.native,
              var y: Double = js.native,
              var z: Double = js.native) extends js.Object

// or simplified - behavior identical

@js.native
@JSImport("three", "Vector3")
class Vector3(var x: Double = js.native) extends js.Object

@js.native
@JSGlobal
class Vector3(var x: Double = js.native) extends js.Object
```

## Output

compilation fails with

```scala
js.native may only be used as stub implementation in facade types
```

## Expectation

Successful compilation, like Scala 2.13.4 does

## Note

#9755 seems related. I'm using Scala 3.0.0-M3 with Scala.js 1.3.1

**Comment by sjrd** [comment]
Thanks for the report.

As a workaround, you can probably declare overloaded constructors.

**Comment by joschua-fink** [comment]
You're welcome! Your suggested workaround compiles and produces plausible JavaScript. For anyone else having this issue, an example that appears to work:

```scala
@js.native
@JSGlobal
class Vector3(var x: Double, 
              var y: Double, 
              var z: Double) extends js.Object {

  def this() =
    this(js.native, js.native, js.native)

}
```

**Comment by sjrd** [comment]
This was fixed in #11857. See also #11592.
